packages/devextreme/scss/widgets/material/popup/_index.scss

```diff
@@ -1,6 +1,4 @@
-@use "sass:color";
 @use "colors" as *;
-@use "../colors" as *;
```

Comment:
Let's leave this import here. It's common practice to use both imports together.
```
@use "colors" as *;
@use "../colors" as *;
```

Reply:
[3fe893f875d19fe5f5c65776c53b7c62a59de559]